XenAdmin/Commands/RestoreHostFromBackupCommand.cs

```diff
@@ -98,10 +98,10 @@ private void Execute(Host host, string filepath)
                     oldDir = Directory.GetCurrentDirectory();
                     OpenFileDialog dialog = new OpenFileDialog();
                     dialog.AddExtension = true;
-                    dialog.Filter = string.Format("{0} (*.xbk)|*.xbk|{1} (*.*)|*.*", Messages.XS_BACKUP_FILES, Messages.ALL_FILES);
+                    dialog.Filter = string.Format("{0} (*." + Branding.BACKUP + ")|*." + Branding.BACKUP + "|{1} (*.*)|*.*", Messages.XS_BACKUP_FILES, Messages.ALL_FILES);

```

Comment:
Instead of string concatenation, we can add Branding.BACKUP as an argument to the Format() function, for example:
`string.Format("{0} (*.{1})|*.{1}|{2} (*.*)|*.*", Messages.XS_BACKUP_FILES, Branding.BACKUP, Messages.ALL_FILES);`